docs/devguide/contributing.asciidoc

```diff
@@ -96,6 +96,8 @@ and documentation. The primary command used for this is:
 --------------------------------------------------------------------------------
 make update
 --------------------------------------------------------------------------------
+Each beat has its own `update` target, as well as a master `update` in the repository root. 
+If a PR adds or removes a dependency, `update` should be run in both the individual beat directory, and the root `beats` directory.
```

Comment:
```suggestion
If a PR adds or removes a dependency, run `update` in both the individual Beat directory and the root `beats` directory.
```

Reply:
This surprises me a bit. I thought you could run `make update` in the root to update everything. What sorts of dependencies require you to run in the individual directories vs the root?

Reply:
Ah, you're actually right. The root `make update` runs its own scripts as well as the individual beats ones.